apps/extension/src/core/managers/transaction.ts

```diff
@@ -111,6 +143,7 @@ class TransactionManager extends BaseManager<Transaction> {
     }
     return txn.outputs
       .map((t) =>
+      isMediaOutput(t) ? t.uri.substring(0, 50) + "..." :
```

Comment:
why do a substring here? we don't truncate text ones. (this is for showing people in the Activity detail pane etc)

Reply:
If I just return the uri, the uri text scrolls infinitely horizontally, as shown below.
<img width="319" alt="image" src="https://github.com/alexanderatallah/window.ai/assets/107221725/43dc7dfa-1a1a-4947-a7a8-b089862c7994">

I added a function that adds a newline every 30 chars, and then chunks the entire uri.
<img width="320" alt="image" src="https://github.com/alexanderatallah/window.ai/assets/107221725/f22aeb07-5abd-4f0a-8fc7-055abd501b7b">
However, the sheer amount of text(ex. the scrollbar is super super long) causes the extension to slow down.

I think a better solution would be to have a link to each object where the user can download their respective file. I have coded this solution up, demo below.

https://github.com/alexanderatallah/window.ai/assets/107221725/19df3753-12a5-4246-8804-d82369b44dc8

Reply:
Cleaned up the code, fully typed it out, and pushed this solution.